README.md

```diff
@@ -200,6 +200,10 @@ OVERLAYS_HOME=/tmp/my-overlays
 
 ```
 
+## Adding jwt auth
+
+This project is thought to work with [jitsi magnify](https://github.com/openfun/jitsi-magnify) so that it can use jwt to authenticate users. To make this possible, change variables in overlay in file `jitsi-common.env` to enable auth, change auth type to jwt, and put app_id and app_secret that will also be put in magnify env.
```

Comment:
Maybe this statement is a bit too strong. Let try a suggestion and feel free to adapt it if I misunderstood:
```suggestion
This project is thought to allow delegating authentication to an external gateway like [jitsi magnify](https://github.com/openfun/jitsi-magnify) which uses JWT tokens to authenticate users. To activate this, you can change variables in the `jitsi-common.env` file of your overlay.  For example, to delegate authentication to [jitsi magnify](https://github.com/openfun/jitsi-magnify), change auth type to jwt, and set values for app_id and app_secret that will also be set to the same values in magnify.
```

Reply:
Yes that's perfect 👍